argmin/src/solver/powell/mod.rs

```diff
@@ -0,0 +1,97 @@
+use crate::core::{
```

Comment:
Minor: Copyright notice is missing

Reply:
Thanks will add this before the PR is final. Leaving this unresolved as a reminder for myself :)